libvast/include/vast/id_range.hpp

```diff
@@ -0,0 +1,27 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2018 The VAST Contributors
```

Comment:
```suggestion
// SPDX-FileCopyrightText: (c) 2022 The VAST Contributors
```

Reply:
This was just copied out of another header to resolve a circular dependency, so I wasn't sure whether to keep the original date or use a new one. I didn't actually change any of the code in this.